client/src/AppRouter.tsx

```diff
@@ -19,6 +20,9 @@ const AppRouter: React.FC = () => {
         {/* Public shared conversation viewer */}
         <Route path="/shared/:shareId" element={<SharedConversationViewer />} />
 
+        {/* Admin page with access code protection */}
+        <Route path="/admin" element={<AdminPage />} />
```

Comment:
please change all admin to qigong because it will manage information relating to qigong but still be protected by access code 

```suggestion
        <Route path="/qigong" element={<AdminPage />} />
```